VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.spec.ts

```diff
@@ -245,4 +249,32 @@ describe('ParticipantStatusComponent', () => {
         component.hearingVenueName = 'venue';
         expect(component.hearingVenueName).toBe('venue');
     });
+
+    describe('setParticipantStatus', () => {
+        hostRoles.forEach(role => {
+            it(`should return "in another hearing text" if participant is a ${role}`, () => {
+                const participant = participants[0];
+                participant.role = role;
+                participant.host_in_another_hearing = true;
+                const participantContactDetails = new ParticipantContactDetails(participant);
+
+                component.setParticipantStatus(ParticipantStatus.InHearing, participantContactDetails);
+
+                expect(participantContactDetails.statusText).toBe(participantStatusReaderSpy.inAnotherHearingText);
+            });
+        });
+
+        hostRoles.forEach(role => {
+            it(`should return get status for ${role} when not in a hearing`, () => {
+                const participant = participants[0];
+                participant.role = role;
+                participant.host_in_another_hearing = false;
+                const participantContactDetails = new ParticipantContactDetails(participant);
+
+                component.setParticipantStatus(ParticipantStatus.InHearing, participantContactDetails);
+
+                expect(participantStatusReaderSpy.getStatusAsTextForHost).toHaveBeenCalled();
```

Comment:
Perhaps give this an and.returnValue and test the statusText is actually set correctly. See line 160